oauth2client/contrib/gce.py

```diff
@@ -105,6 +73,8 @@ def __init__(self, scope='', **kwargs):
         # Assertion type is no longer used, but still in the
         # parent class signature.
         super(AppAssertionCredentials, self).__init__(None)
+
+        # Cache until Metadata Server supports Cache-Control Header
```

Comment:
Seems like this is kind of a given. I feel like regardless of it the metadata server supported Cache-Control that this class should only ask for the service account email once.